api/nakadi-event-bus-api.yaml

```diff
@@ -1191,11 +1191,6 @@ definitions:
       - $ref: '#/definitions/Cursor'
       - type: object
         properties:
-          id:
-            type: string
-            format: uuid
-            description: |
-              Id of the cursor (a UUID).
           event_type_name:
```

Comment:
Sure. I can do it in a separate PR to keep the scope of changes really small so we can move fast.